src/js/schemas/service-schema/EnvironmentVariables.js

```diff
@@ -1,14 +1,16 @@
-import {Link} from 'react-router';
 import {Hooks} from 'PluginSDK';
 /* eslint-disable no-unused-vars */
 import React from 'react';
 /* eslint-enable no-unused-vars */
 
 let EnvironmentVariables = {
-  description: (
-    <span>
-      Set variables for each task your service launches. You can also use variables to expose Secrets. <Link to="system-security-secrets">Manage secrets here</Link>. <a href="https://mesosphere.github.io/marathon/docs/task-environment-vars.html" target="_blank">Learn more about variables</a>.
-    </span>  
+  description: Hooks.applyFilter(
```

Comment:
going to create this hook here just incase plugins need to change the description